examples/boa/boa.go

```diff
@@ -1089,11 +978,12 @@ func (ss *Sim) Log(mode etime.Modes, time etime.Times) {
 		// 	ss.Loops.Stop(etime.Trial)
 		// }
 
-		// trnEpc := ss.Loops.Stacks[etime.Train].Loops[etime.Epoch].Counter.Cur
-		// if ss.StopOnErr && trnEpc > 5 && ev.Rew < 0 {
-		// 	fmt.Printf("STOPPED due to negative reward for US: %d  %g\n", ev.US, ev.Rew)
-		// 	ss.Loops.Stop(etime.Trial)
-		// }
+		ev := TheSim.Envs[etime.Train.String()].(*Approach)
```

Comment:
comment this out before committing?

Reply:
it is harmless actually unless you hit the StopOnErr button and this is as good an error to check as any at this point..